CONTRIBUTING.md

```diff
@@ -58,7 +58,7 @@ We're following the [Google Go Code Review](https://code.google.com/p/go-wiki/wi
 + Once ready to land your change, squash your commits. Where n is the number of commits in your branch, run
 `git rebase -i HEAD~n`
 
- and subsequently update your remote (you will have to force the push, `git push -f andybons mybranch`). The pull request will update.
+ and subsequently update your remote (you will have to force the push, `git push -f $USER mybranch`). The pull request will update.
```

Comment:
On second thought, since this file is for outside contributors we should probably rework the instructions to use a fork. (you can't follow these instructions until you've been given push access to the main repo). But that can be a separate change.